### ISSUE TITLE
Bad color contrast for failed validation fields on user form in dark mode

### ISSUE THREAD

**Issue by MasterOdin** [Submitty/Submitty#5541]
**Describe the bug**
Fields that fail validation have a pink background and white text on dark mode. The text is not legible.

**Expected behavior**
The test to be legible.

**To Reproduce**
1. Open a course page
2. Go to Users page
3. Edit user and type "Test!!!" into user firstname

**Configuration**
 - OS: [e.g. Windows, Linux, ...] macOS
 - Browser [e.g. chrome, safari] Chrome

**Screenshots**

![Screenshot 2020-06-16 18 17 10](https://user-images.githubusercontent.com/1845314/84833685-85f05a00-b038-11ea-8116-f8b19b5a96a5.png)
